src/SUMMARY.md

```diff
@@ -65,6 +65,7 @@
     - [Beta Backporting](./release/beta-backporting.md)
     - [Platform Support](./release/platform-support.md)
     - [Preparing Release Notes](./release/release-notes.md)
+    - [Priority Levels](./release/priority-levels.md)
```

Comment:
For what it's worth, my vote goes to placing this under the "Triage Procedure" chapter as I see it more relevant to that.

Or somehow rework the index and promote the triaging process to its own chapter hierarchy

Reply:
I'd say this needs to go on Compiler -> Prioritization -> Priority Levels

Reply:
Just moved it, and created a Compiler/Prioritization page.

Any thoughts on what we woud like to write in this page ?

Reply:
Compiler/Prioritization would need to have at some point our procedure. Basically this https://github.com/rust-lang/compiler-team-prioritization/issues/6

Reply:
Ohh sorry, the procedure should go in a sibling section. In the top level we may want to put something like ...

> This section documents the processes of the prioritization WG.

Similar to https://forge.rust-lang.org/community/index.html